AppCenter/AppCenter/Internals/Channel/MSChannelUnitConfiguration.m

```diff
@@ -20,8 +20,12 @@ - (instancetype)initWithGroupId:(NSString *)groupId
   return self;
 }
 
+- (instancetype)initWithGroupId:(NSString *)groupId flushInterval:(NSUInteger)flushInterval {
```

Comment:
It still uses default configurations except flush interval. I guess it is good to keep the name.
```suggestion
- (instancetype)initDefaultConfigurationWithGroupId:(NSString *)groupId flushInterval:(NSUInteger)flushInterval {
```